uwp/Pdf-Viewer/Concepts-and-Features/Working-with-free-text-callout-annotation.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: Free text calllout annotation in UWP PDF Viewer control | Syncfusion
+description: Learn here all about free text callout annotation support in Syncfusion UWP PDF Viewer (SfPdfViewer) control and more.
+platform: UWP
+control: SfPdfViewerControl
+documentation: ug
+---
+
+# Working with free text callout annotations in UWP PDF Viewer (SfPdfViewer)
+
+PDF Viewer allows you to include free text callout annotations in a PDF document and provides options to modify or remove the existing free text callout annotations.
+
+## Adding free text callout annotations
+
+### Enabling free text callout annotation mode
+
+To enable a free text callout annotation, execute the `FreeTextCalloutAnnotationCommand` with true as parameter.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfPdfViewerControl x:Name="pdfViewer"/>
+<Button x:Name="FreeTextCalloutAnnotationButton" Click="FreeTextCalloutAnnotationButton_Click"/>
+
+{% endhighlight %}
+{% highlight c# %}
+
+private void FreeTextCalloutAnnotationButton_Click(object sender, RoutedEventArgs e)
+{
+	pdfViewer.FreeTextCalloutAnnotationCommand.Execute(true);
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Disabling free text callout annotation mode
+
+Free text callout annotation can be disabled by executing the same command with false as parameter.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfPdfViewerControl x:Name="pdfViewer"/>
+<Button x:Name="ResetAnnotationButton" Click="ResetAnnotationButton_Click" />
+
+{% endhighlight %}
+{% highlight c# %}
+
+private void ResetAnnotationButton_Click(object sender, RoutedEventArgs e)
+{
+	pdfViewer.FreeTextCalloutAnnotationCommand.Execute(false);
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the appearance of free text callout annotations
+
+You can customize the default values of text color, text size, stroke color, fill color, stroke width and opacity of all free text callout annotations to be added. This will not affect the already added free text annotations.
```

Comment:
You can customize the default values of text color, text size, stroke color, fill color, stroke width, and opacity of all free text callout annotations to be added. This will not affect the already added free text annotations.